tracer/src/Datadog.Trace.Coverage.collector/CoverageCollector.cs

```diff
@@ -162,33 +162,54 @@ internal void ProcessFolder(string folder, SearchOption searchOption)
                     {
                         if (File.Exists(Path.Combine(path, fileWithoutExtension + ".pdb")) || File.Exists(Path.Combine(path, fileWithoutExtension + ".PDB")))
                         {
-                            try
+                            List<Exception>? lstExceptions = null;
```

Comment:
HUNGARIAN NOTIATION!? How _very_ dare you 😂 😛 
```suggestion
                            List<Exception>? exceptions = null;
```